.github/workflows/go.yml

```diff
@@ -1,13 +1,30 @@
 name: Go
 on: [push]
```

Comment:
Last request. Change it to `on: [push, pull_request]` to test your settings before merge.

Reply:
I have changed it to that in the latest commit.